stan/math/opencl/kernel_generator/elt_function_cl.hpp

```diff
@@ -302,6 +302,7 @@ ADD_UNARY_FUNCTION_WITH_INCLUDES(inv_logit,
 ADD_UNARY_FUNCTION_WITH_INCLUDES(logit, opencl_kernels::logit_device_function)
 ADD_UNARY_FUNCTION_WITH_INCLUDES(
     log1m_inv_logit, opencl_kernels::log1m_inv_logit_device_function)
+ADD_UNARY_FUNCTION_WITH_INCLUDES(square, "double square(double x){return x*x;}")
```

Comment:
Optional:

I would maybe rather see this one in a separate file, so we are consistent in device function code location.

Reply:
It is so trivial I doubt it will ever be used outside of kernel generator, so I just put it here.

Reply:
Ok, lets leave it then.